docs/tutorials/demo/Alchemiscale Demo.ipynb

```diff
@@ -0,0 +1,2774 @@
+      "    comment: Table 4, entry 48\n",
+      "    doi: 10.1016/j.ejmech.2013.03.070\n",
+      "    error: 0.072\n",
+      "    type: ki\n",
+      "    unit: uM\n",
+      "    value: 0.24\n",
+      "  name: lig_ejm_48\n",
+      "  smiles: '[H]c1c(c(c(c(c1[H])Cl)C(=O)N([H])c2c(c(nc(c2[H])N([H])C(=O)C3(C(C(C(C3([H])[H])([H])[H])([H])[H])([H])[H])[H])[H])[H])Cl)[H]'\n",
+      "lig_ejm_50:\n",
+      "  measurement:\n",
+      "    comment: Table 4, entry 50\n",
+      "    doi: 10.1016/j.ejmech.2013.03.070\n",
+      "    error: 0.075\n",
+      "    type: ki\n",
+      "    unit: uM\n",
+      "    value: 0.25\n",
+      "  name: lig_ejm_50\n",
+      "  smiles: '[H]c1c(c(c(c(c1[H])Cl)C(=O)N([H])c2c(c(nc(c2[H])N([H])C(=O)C([H])([H])O[H])[H])[H])Cl)[H]'\n",
+      "lig_ejm_54:\n",
+      "  measurement:\n",
+      "    comment: Table 4, entry 54\n",
+      "    doi: 10.1016/j.ejmech.2013.03.070\n",
+      "    error: 0.0054\n",
+      "    type: ki\n",
+      "    unit: uM\n",
+      "    value: 0.018\n",
+      "  name: lig_ejm_54\n",
+      "  smiles: '[H]c1c(c(c(c(c1[H])Cl)C(=O)N([H])c2c(c(nc(c2[H])N([H])C(=O)N([H])C([H])([H])C([H])([H])[H])[H])[H])Cl)[H]'\n",
+      "lig_ejm_55:\n",
+      "  measurement:\n",
+      "    comment: Table 4, entry 55\n",
+      "    doi: 10.1016/j.ejmech.2013.03.070\n",
+      "    error: 0.051\n",
+      "    type: ki\n",
+      "    unit: uM\n",
+      "    value: 0.17\n",
+      "  name: lig_ejm_55\n",
+      "  smiles: '[H]c1c(c(c(c(c1[H])Cl)C(=O)N([H])c2c(c(nc(c2[H])N([H])C(=O)OC([H])([H])[H])[H])[H])Cl)[H]'\n",
+      "lig_jmc_23:\n",
+      "  measurement:\n",
+      "    comment: Table 2, entry 23; values for four different enantionmers listed, this\n",
+      "      one is for cis\n",
+      "    doi: 10.1021/jm400266t\n",
+      "    error: 0.75\n",
+      "    type: ki\n",
+      "    unit: nM\n",
+      "    value: 2.5\n",
+      "  name: lig_jmc_23\n",
+      "  smiles: '[H]c1c(c(c(c(c1[H])Cl)C(=O)N([H])c2c(c(nc(c2[H])N([H])C(=O)[C@@]3([C@](C3([H])[H])([H])F)[H])[H])[H])Cl)[H]'\n",
+      "lig_jmc_27:\n",
+      "  measurement:\n",
+      "    comment: Table 2, entry 27; cis enantiomer\n",
+      "    doi: 10.1021/jm400266t\n",
+      "    error: 1.5\n",
+      "    type: ki\n",
+      "    unit: nM\n",
+      "    value: 5.1\n",
+      "  name: lig_jmc_27\n",
+      "  smiles: '[H]c1c(c(c(c(c1[H])Cl)C(=O)N([H])c2c(c(nc(c2[H])N([H])C(=O)[C@@]3([C@](C3([H])[H])([H])Cl)[H])[H])[H])Cl)[H]'\n",
+      "lig_jmc_28:\n",
+      "  measurement:\n",
+      "    comment: Table 2, entry 28; cis enantiomer\n",
+      "    doi: 10.1021/jm400266t\n",
+      "    error: 2.6\n",
+      "    type: ki\n",
+      "    unit: nM\n",
+      "    value: 8.5\n",
+      "  name: lig_jmc_28\n",
+      "  smiles: '[H]c1c(c(c(c(c1[H])Cl)C(=O)N([H])c2c(c(nc(c2[H])N([H])C(=O)[C@@]3([C@](C3([H])[H])([H])C([H])([H])[H])[H])[H])[H])Cl)[H]'\n"
+     ]
+    }
+   ],
+   "source": [
+    "%cat ligands.yml"
```

Comment:
You don't really explain what's in this file here? Probably something like "we have a PLB yaml file with experimental data for each ligand in our network"

Reply:
Added clarification!